elastic/datadog_checks/elastic/data/conf.yaml.example

```diff
@@ -22,20 +22,63 @@ instances:
     #
     # password: <PASSWORD>
 
-    ## @param ssl_verify - boolean - optional - default: false
-    ## Instruct the check to validate SSL certificates when connecting to "url"
+    ## @param tls_verify - boolean - optional - default: true
```

Comment:
Would it be possible to re-use descriptions wording from: https://github.com/DataDog/integrations-core/pull/4094/files#diff-a3a33418edc5df18cad3c0fd475ba075R200 in order to be consistent after the small wordings updates that were applied on this PR ? :)